airflow/www/templates/airflow/graph.html

```diff
@@ -683,10 +683,18 @@
           source_id = map_task_to_node.get(edge.source_id)
           target_id = map_task_to_node.get(edge.target_id)
           if(source_id != target_id && !g.hasEdge(source_id, target_id))
-            g.setEdge(source_id, target_id, {
-              curve: d3.curveBasis,
-              arrowheadClass: 'arrowhead',
-            });
+            if (edge.label != undefined) {
+              g.setEdge(source_id, target_id, {
+                curve: d3.curveBasis,
+                arrowheadClass: 'arrowhead',
+                label: edge.label
+              });
+            } else {
+              g.setEdge(source_id, target_id, {
+                curve: d3.curveBasis,
+                arrowheadClass: 'arrowhead',
+              });
+            }
```

Comment:
This should be fine without the `undefined` check:
```suggestion
            g.setEdge(source_id, target_id, {
              curve: d3.curveBasis,
              arrowheadClass: 'arrowhead',
              label: edge.label,
            });
```

Reply:
👍 Wasn't sure if I could, but great, I'll do that!